src/components/PersonLink.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  people: Person[]
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slugId } = useParams();
+  const findMother = people.find(
```

Comment:
FindMother is sounds as a function, but it's a variable 
So, I suggest you to rename to it to foundMother or matchedMother